### PR TITLE
Enabling openldap replication 

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
- Problem statement 

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/setup/openldap/openldap.py [hunk 1]
```diff
@@ -89,10 +89,10 @@ def _key_value_verify(self, key: str, phase: str):
             Log.debug("Validation failed for %s in %s phase\n" % (key ,phase))
             raise Exception("Validation failed for %s in %s phase" % (key ,phase))
         else:
-            if (("cortx>software>openldap>base_dn" == key) and (not bool(re.match("^dc=[a-zA-Z0-9]+,dc=[a-zA-Z0-9]+$", value)))):
+            if ((Conf.get(self.prov, 'CONFIG>OPENLDAP_BASE_DN') == key) and (not bool(re.match("^dc=[a-zA-Z0-9]+,dc=[a-zA-Z0-9]+$", value)))):
```
Comment (abhilekh268) [inline py-utils/src/utils/setup/openldap/openldap.py:92]: we should not fix this regex matching to only two dc...

Comment (shahidSTX) [reply py-utils/src/utils/setup/openldap/openldap.py:92]: Yes, I'm also of the opinion that we should have no validation instead of too stern validation.

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/openldap.py:92]: @bkirunge7 plz comment here

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/openldap.py:92]: regex already in branch. Here I am just chaging hardcoding.
So if you suggest then I will remove the regex validation from merged code

**Review by bkirunge7 — APPROVED** [review]

File: py-utils/src/utils/setup/openldap/openldap.py (hunk 1 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/openldap.py]: @abhilekh268  handled regex . Now it can handle multiple dcs too. plz re-check

**Review by abhilekh268 — APPROVED** [review]

**Pull request merged**